iosxr_setup.py

```diff
@@ -1,18 +1,21 @@
 '''
 Author: Rich Wellum (richwellum@gmail.com)
 
-Login to IOS XR, apply some config and then exit.
+Login to a running IOS XR VM, then apply a combination of XR and XR linux
```

Comment:
@glennmatthews Fixed your comments. Retested both k9 and non-k9 images and works well.